main.go

```diff
@@ -81,21 +81,37 @@ func main() {
 					Usage:  "URL which receives a POST on state change",
 					EnvVar: "SHAWARMA_URL",
 				},
+				cli.StringFlag{
+					Name:   "state-notifier, s",
+					Value:  "true",
+					Usage:  "Enable/Disable state change notification",
+					EnvVar: "SHAWARMA_STATE_NOTIFIER",
+				},
+				cli.StringFlag{
```

Comment:
about this case, the http server handler expects a string with the hostname to bind, but also I was thinking about Kubernetes [envs](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.19/#envvar-v1-core) which are explicitly required to be strings and I want to avoid unnecessary type casting, what do you think? can we keep it as string?

Reply:
I think it makes sense to have the command line arguments handled as numbers when they must be numbers. That way we're hitting the validation logic in the command line parser and giving meaningful error messages if the value supplied is incorrect.

Yes, this will result in some extra conversions from env var to integer and back to string on the HTTP server. But this is very inexpensive overall when you consider it's only happening once at startup. To me the cost is worth it.

Reply:
👍  updated the cli type to be expected as IntFlag then treated as string for HTTP server functions, already tested from CLI and k8s environments without any issues. Thanks